packages/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -241,14 +242,14 @@ export class TriggerEvent {
               channelType
             );
             if (provider) {
-              providers.set(channelType, provider);
+              providers[channelType] = provider;
```

Comment:
Isn't this to cause problems for example in the email channel with the Novu provider plus the email provider a user might enable?

Reply:
and won't return the novu email integration?